tests/contenttypes_tests/tests.py

```diff
@@ -378,6 +378,14 @@ class InvalidBookmark(models.Model):
         self.assertEqual(errors, expected)
 
 
+class GenericRelationshipModelTests(TestCase):
+    def test_value_to_string(self):
+        q = Question.objects.create(text='test')
+        Answer.objects.create(question=q)
+        Answer.objects.create(question=q)
+        self.assertEqual(Question.answer_set.field.value_to_string(q), '[1, 2]')
```

Comment:
Don't use hard coded IDs. Didn't the first version of your patch fail because of that?